source/optional_features/custom_error_pages/index.html.md.erb

```diff
@@ -0,0 +1,24 @@
+---
+title: Custom error pages 
+weight: 135
+---
+
+# Custom error pages
+
+GOV.UK Pay supports custom error pages.
+
+When a payment reaches a terminal, unsuccessful state, GOV.UK Pay will
+immediately redirect the user to the return URL (`return_url`), which you can
+set in the response to the `POST /v1/payments` API call. 
+
+To set this up for your service, you should make a query to the GOV.UK Pay API
+to determine what the failure state is, and set an appropriate error message. 
+
+If you do not use custom error pages, users will see generic unsuccessful
```

Comment:
not sure about using 'generic' here - it could suggest that Pay only has one generic unsuccessful payment screen.

Reply:
From Alex: want to get across that the pages are "generic" in the sense that, regardless of what your service is you'll have similar error pages — if the service does their own error pages, they're in a good place to tell the user what to do with a specific problem. 

Could use the term "standard GOV.UK Pay error pages" — these are generally good, but services can make them more specific to the problem(s) at hand, depending on circumstances.

Reply:
'standard' works; see my comment about 'error' above